src/main/java/heronarts/lx/midi/surface/MidiFighterTwister.java

```diff
@@ -337,12 +350,57 @@ public void onParameterChanged(LXParameter parameter) {
       }
     }
 
+    private boolean isKnobRelative(int index) {
+      // Fixed for now but could be expanded to poll MFT encoder settings on startup with sysex
+      return true;
+    }
+
     private void onKnob(int index, double normalized) {
       if (this.knobs[index] != null) {
         this.knobs[index].setNormalized(normalized);
       }
     }
 
+    private final static double KNOB_INCREMENT_AMOUNT = 1.0/128.0;
+
+    private void onKnobDecrement(int index) {
+      LXListenableNormalizedParameter knob = this.knobs[index];
+      if (knob != null) {
+        if (knob instanceof DiscreteParameter) {
+          // Move after a set number of ticks in the same direction
+          this.knobTicks[index] = LXUtils.min(this.knobTicks[index], 0) - 1;
+          if (this.knobTicks[index] == 0-KNOB_TICKS_PER_DISCRETE_INCREMENT) {
```

Comment:
`0-KNOB_TICKS_PER_DISCRETE_INCREMENT` feels weird, just put `-KNOB_TICKS_PER_DISCRETE_INCREMENT` without the leading 0

Reply:
Got it, done.